packages/chart/src/ChartUtils.ts

```diff
@@ -1908,8 +1929,21 @@ class ChartUtils {
         yaxis: this.makeLayoutAxis(dh.plot.AxisType.Y, theme),
         zaxis: this.makeLayoutAxis(dh.plot.AxisType.Z, theme),
       },
+      geo: {
+        showcoastlines: true,
```

Comment:
Need to add `geo.bgcolor`. I'd say just set it to `plot_bgcolor` instead of making a separate variable for it